cps/pom.xml

```diff
@@ -77,13 +77,35 @@
             <artifactId>groovy-cps</artifactId>
             <version>1.5</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.ui</groupId>
+            <artifactId>jquery-detached</artifactId>
+            <version>1.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.ui</groupId>
+            <artifactId>ace-editor</artifactId>
+            <version>1.0</version>
+        </dependency>
     </dependencies>
     <!-- TODO currently job depends on tests from here; this dependency should probably be broken -->
     <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+            </resource>
+            <resource>
+                <directory>target/generated-adjuncts</directory>
+            </resource>
+        </resources>
         <plugins>
             <plugin>
                 <artifactId>maven-jar-plugin</artifactId>
                 <executions>
+                    <!-- 
+                        TODO currently job depends on tests from here; this dependency should probably be broken.
+                        TODO Update the above TODO to make it clear what it's talking about. By "job", I think it is referring to the "workflow-job" module. 
```

Comment:
:ant: I meant to _move_ the existing comment, not _copy_ it. `job` refers to the `job` submodule (`workflow-job` plugin).